### PR TITLE
fix: fix examples for layout components

### PR DESCRIPTION
**Pull request by burakukula**

### CODE REVIEW

File: packages/core/src/Box/examples/BoxPolymorphicExample.tsx [hunk 1]
```diff
@@ -7,15 +7,6 @@ export default function BoxSpacingExample() {
       <Box as="span" display="inline" marginRight="spacingL">
         I am span
       </Box>
-      <Box
-        as="a"
```
Comment (burakukula) [inline packages/core/src/Box/examples/BoxPolymorphicExample.tsx:11]: Too controversial 😄. We shouldn’t recommend using Box to create links, users should use `TextLink`.

File: packages/core/src/Flex/examples/FlexExample.tsx [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
-  );
-
   return (
     <Flex justifyContent="space-between" alignItems="center">
-      <ColorBox backgroundColor={tokens.green500}>Green</ColorBox>
-      <ColorBox backgroundColor={tokens.red500}>Red</ColorBox>
-      <ColorBox backgroundColor={tokens.blue500}>Blue</ColorBox>
+      <Box
+        justifyContent="center"
+        alignItems="center"
```
Comment (bgutsol) [inline packages/core/src/Flex/examples/FlexExample.tsx:10]: `Box` doesn’t have flex styles, same comment for all other places
```suggestion
```

**Review by bgutsol — CHANGES_REQUESTED** [review]

File: packages/core/src/Flex/examples/FlexExample.tsx (hunk 2 above)
Comment (suevalov) [reply packages/core/src/Flex/examples/FlexExample.tsx:10]: Exactly, I'd keep ColorBox component here, but we can use `<div>` and center text differentely

File: packages/core/src/Stack/examples/VerticalStackExample.tsx [hunk 3]
```diff
@@ -1,28 +1,40 @@
 import React from 'react';
-import { Stack, Flex } from '@contentful/f36-components';
+import { Stack, Box } from '@contentful/f36-components';
 import tokens from '@contentful/f36-tokens';
 
 export default function VerticalStackExample() {
-  const ColorBox = ({ backgroundColor, children }) => (
```
Comment (suevalov) [inline packages/core/src/Stack/examples/VerticalStackExample.tsx:6]: I think having a component here is actually good and it allows to understand `<Stack>` code easier without stealing too much attention to children.

**Review by suevalov — CHANGES_REQUESTED** [review]

File: packages/core/src/Box/examples/BoxPolymorphicExample.tsx (hunk 1 above)
Comment (suevalov) [reply packages/core/src/Box/examples/BoxPolymorphicExample.tsx:11]: Maybe we need to replace this example with something else to keep 3 items in a row?

File: packages/core/src/Flex/examples/FlexExample.tsx (hunk 2 above)
Comment (burakukula) [reply packages/core/src/Flex/examples/FlexExample.tsx:10]: sure, overlooked. sorry. 
But I think we don't need text inside boxes at all.

File: packages/core/src/Stack/examples/VerticalStackExample.tsx (hunk 3 above)
Comment (burakukula) [reply packages/core/src/Stack/examples/VerticalStackExample.tsx:6]: Well, I don't really agree. The only code to understand here is `<Stack flexDirection="column" spacing="spacingS">`. In other examples, for Flex or Grid, we use Boxes directly. I think it is actually not needed to have this abstraction here and it makes it more consistent with other docs.

File: packages/core/src/Box/examples/BoxPolymorphicExample.tsx (hunk 1 above)
Comment (burakukula) [reply packages/core/src/Box/examples/BoxPolymorphicExample.tsx:11]: Why do we need 3 items?

Comment (suevalov) [reply packages/core/src/Box/examples/BoxPolymorphicExample.tsx:11]: just to have more than 2 examples :)

**Review by suevalov — APPROVED** [review]

**Pull request closed without merging**